Milliner/cfg/cfg.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+return [
+    'fedora base url' => 'http://localhost:8080/fcrepo/rest',
+    'drupal base url' => 'http://localhost:8000',
```

Comment:
would it be possible to avoid spaces in these array keys? (just thinking about the future YAML-based config...)